configs/x86/system_definition.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+system_definition:
+  name: x86
+  manufacturer:
+    vendor: ""
+    name: ""
+  processor:
+    vendor: ""
+    name: ""
+    ISA: x86_64
+  accelerator:
+    vendor: ""
+    name: ""
+    GPU_ISA: ""
+  interconnect:
+    vendor: ""
+    name: ""
+  OS:
+    name: ""
+  system-tested:
+    owner: ""
+    name: ""
+    installation-year: ""
+    description: ""
```

Comment:
Is the idea this eventually gets the information for the cloud node we use to test this on?

Reply:
probably?  A cloud node could have the specific cluster type (which defines hardware) and the specific cluster build (which defines software stack), but I am not sure which goes here since for other clusters we only define the hardware in this file (and the OS unless we remove it).